pkgs/applications/editors/neovim/update-treesitter-parsers.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i python3 -p python3
+
+import re
+import subprocess
+from pathlib import Path
+
+parsers = {}
+dir = Path(__file__).parent
+regex = re.compile(r"^set\(TREESITTER_([A-Z_]+)_(URL|SHA256)\s+([^ \)]+)\s*\)\s*$")
+
+src = subprocess.check_output(
+    [
+        "nix-build",
+        dir.parent.parent.parent.parent,
+        "-A",
+        "neovim-unwrapped.src",
+        "--no-out-link",
+    ],
+    text=True,
+).strip()
+
+for line in open(f"{src}/cmake.deps/CMakeLists.txt"):
```

Comment:
a comment for potential other reviewers, neovim may soon put the versions in a .json file easier to parse.